sheetstorm-deployment/roles/configure_nginx/templates/sheetstorm-remote.j2

```diff
@@ -47,6 +56,11 @@ server {
     include /etc/letsencrypt/options-ssl-nginx.conf; # managed by Certbot
     ssl_dhparam /etc/letsencrypt/ssl-dhparams.pem; # managed by Certbot
 
+    # Enable HSTS header
+    add_header Strict-Transport-Security "max-age=31536000; includeSubdomains; preload";
+
+    # Add Content Security Policy header that defines from which sources, browser can download content.
+    add_header Content-Security-Policy "base-uri 'self'; form-action 'self'; default-src 'none'; script-src 'self' {% for source in content_security_policy_header.allow_script_sources %}{{ source }} {% endfor %}; connect-src 'self'; img-src 'self'; font-src 'self' {% for source in content_security_policy_header.allow_font_sources %}{{ source }} {% endfor %}; style-src 'self' {% for source in content_security_policy_header.allow_stylesheet_sources %}{{ source }} {% endfor %}; object-src 'none'; frame-ancestors 'none';" always;
```

Comment:
You add to `sheetstorm-deployment/roles/configure_nginx/templates/sheetstorm-remote.j2` and `sheetstorm-deployment/roles/configure_nginx/templates/maintenance-remote.j2` same things. I'm not an expert in deployment things but can't you extract it to somewhere where these two files can import from?

Reply:
This should be consider as separate pull request with refactor.